lib/src/models/models.dart

```diff
@@ -569,10 +569,22 @@ abstract class AbstractControl<T> {
     }
   }
 
-  /// Returns true if all children disabled, otherwise returns false.
+  /// Returns true if all children are disabled, otherwise returns false.
   @protected
   bool allControlsDisabled() => disabled;
 
+  /// Returns true if all children are touched, otherwise returns false.
+  @protected
+  bool allControlsTouched() => touched;
+
+  /// Returns true if all children are dirty, otherwise returns false.
+  @protected
+  bool allControlsDirty() => dirty;
+
+  /// Returns true if all children are pristing, otherwise returns false.
+  @protected
+  bool allControlsPristine() => pristine;
```

Comment:
Hi @vasilich6107,

Nice work in all of the MR. In this particular MR, I have some doubts. 
Do we really need all these methods? I mean the **allControlsDisabled** is required because it is used to calculate the status of the FormGroup and FormArray depending if all controls are disabled or not. But the others methods that you included are not used to calculate anything, and I'm not sure a package consumer will use them.

Do you have any real use case in which these methods are used?